en/cocos-service/sdkhub-plugins/sdkhub-hms.md

```diff
@@ -195,6 +240,27 @@ sdkhub.getUserPlugin().callFuncWithParam("checkAppUpdate");
 | + 132 | JSON | CheckAppUpdate successfully, The returned information can correspond to [intent](https://developer.huawei.com/consumer/en/doc/HMSCore-References-V5/appupdateclient-0000001050123641-V5#EN-US_TOPIC_0000001054371620__section15712187193218) |
 | + 107 | JSON / String | Operation failure description |
 
+#### `accountLogin`
+
+**Note**: Game does not need to integrate this method.
+
+This method enable applications of developers to obtain users' authorization codes (temporary authorization credentials) or ID tokens (identity authorization information), so that users can sign in with their HUAWEI IDs securely. Please refer to [Signing In with a HUAWEI ID](https://developer.huawei.com/consumer/en/doc/development/HMSCore-Guides/dev-guide-account-0000001050048888#ZH-CN_TOPIC_0000001050048888__section15992612272) and [Silently Signing In with a HUAWEI ID](https://developer.huawei.com/consumer/en/doc/development/HMSCore-Guides/login-silentsignin-0000001050050853) documents.
```

Comment:
This method is used by third-party applications to obtain the user authentication information (ID Token) or the user's temporary authorization ticket (Authorization Code) of Huawei account, so that the user can use the Huawei account to securely log in to third-party applications. For details, please refer to the [Signing In with a HUAWEI ID](https://developer.huawei.com/consumer/en/doc/development/HMSCore-Guides/dev-guide-account-0000001050048888#ZH-CN_TOPIC_0000001050048888__section15992612272) and [Silently Signing In with a HUAWEI ID](https://developer.huawei.com/consumer/en/doc/development/HMSCore-Guides/login-silentsignin-0000001050050853) documentations.

Reply:
已修改